Makefile.vendor.mk

```diff
@@ -1,7 +1,7 @@
-VERSION = 3.0.0-tp.1
+VERSION = 3.0.0-tp.2
 OPERATOR_NAME = servicemeshoperator3
 HUB = quay.io/maistra-dev
 CHANNELS = candidates
 HELM_VALUES_FILE = ossm/values.yaml
-VERSIONS_YAML_FILE = versions.yaml
+VERSIONS_YAML_FILE ?= ossm/versions.yaml
```

Comment:
The whole purpose of this variable is to make it possible to use another versions file without changing the Makefile. Just set `export VERSIONS_YAML_FILE=ossm/versions.yaml` before invoking make and you're all set.

Reply:
yes but this vendor definition is applied before we do the ?= in the core Makefile so this change is correct - otherwise it wouldn't be possible to overwrite in env var anymore (which @FilipB is doing in his other PR)

Reply:
This is a change in default behavior. Now `make gen` will use `ossm/versions.yaml`. So to run e2e tests against all versions in `./versions.yaml`, we will have to do: